common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -5734,6 +5734,26 @@ public static enum ConfVars {
             + "a background update happens periodically to report the actual state of the query"),
     HIVE_SCHEDULED_QUERIES_CREATE_AS_ENABLED("hive.scheduled.queries.create.as.enabled", true,
         "This option sets the default behaviour of newly created scheduled queries."),
+    HIVE_QUERY_HISTORY_SERVICE_ENABLED("hive.query.history.service.enabled", true,
```

Comment:
I don't think it should be enabled by default as it affects the query performance (`hive.query.history.service.explain.plan` requires additional processing)

Reply:
hm, right, I'm afraid if I want to set hive.query.history.service.explain.plan.enabled=true by default (which is the expected use-case), that's another reason to disable this service by default

Reply:
probably that is something we should to understand the impact and defaults setup